homeassistant/components/google_assistant/helpers.py

```diff
@@ -141,18 +161,32 @@ def async_schedule_google_sync(self, agent_user_id: str):
             self.hass, SYNC_DELAY, _schedule_callback
         )
 
+    @callback
+    def async_schedule_google_sync_all(self):
+        """Schedule a sync for all registered agents."""
+        for agent_user_id in self._store.agent_user_ids:
+            self.async_schedule_google_sync(agent_user_id)
```

Comment:
We don't have to keep `async_schedule_google_sync` around, we will never want a single sync to be done ? Probably same for a bunch of other methods that involve a single action instead of doing it for all connected users

Reply:
nevermind, realize now that the per-agent-user-id methods are implemented by the implementations and not the base class.